sdk/metric/controller/push/push.go

```diff
@@ -28,20 +28,21 @@ import (
 
 // Controller organizes a periodic push of metric data.
 type Controller struct {
-	lock         sync.Mutex
-	collectLock  sync.Mutex
-	accumulator  *sdk.Accumulator
-	resource     *resource.Resource
-	uniq         metric.MeterImpl
-	named        map[string]metric.Meter
-	errorHandler sdk.ErrorHandler
-	integrator   export.Integrator
-	exporter     export.Exporter
-	wg           sync.WaitGroup
-	ch           chan struct{}
-	period       time.Duration
```

Comment:
"period" should not be needed any more.

Reply:
Also, is the formatting done automatically by "gofmt"? Please check opentelemetry's repository to see if it has its own formatting config, if yes, make sure we are using that.

Reply:
Good catch!